File: .java/Program/PetShop.java
public class PetShop {
    private int ID;
    private String nama;
    private int harga;
    private int stok;

    public PetShop() {
        this.ID = 0;
        this.nama = "";
        this.harga = 0;
        this.stok = 0;
    }

    public PetShop(int ID, String nama, int harga, int stok) {
        this.ID = ID;
        this.nama = nama;
        this.harga = harga;
        this.stok = stok;
    }

    public void setID(int ID) { this.ID = ID; }
    public int getID() { return this.ID; }

    public void setNama(String nama) { this.nama = nama; }
    public String getNama() { return this.nama; }

    public void setHarga(int harga) { this.harga = harga; }
    public int getHarga() { return this.harga; }

    public void setStok(int stok) { this.stok = stok; }
    public int getStok() { return this.stok; }
}

File: .cpp/Program/PetShop.cpp
#include <bits/stdc++.h>
using namespace std;

class PetShop {
private:
    int ID;
    string nama;
    int harga;
    int stok;

public:
    PetShop () {
        ID = 0;
        nama = "";
        harga = 0;
        stok = 0;
    };

    PetShop (int ID, string nama, int harga, int stok) {
        this->ID = ID;
        this->nama = nama;
        this->harga = harga;
        this->stok = stok;
    }

    void setID(int ID) { this->ID = ID; }
    int getID() { return this->ID; }

    void setNama(const string& nama) { this->nama = nama; }
    string getNama() { return this->nama; }
    
    void setHarga(int harga) { this->harga = harga; }
    int getHarga() { return this->harga; }

    void setStok(const int stok) { this->stok = stok; }
    int getStok() { return this->stok; }

    ~PetShop () { }
};
